reference/constraints/NegativeOrZero.rst

```diff
@@ -0,0 +1,83 @@
+NegativeOrZero
+==============
+
+.. versionadded:: 4.3
+
+    The ``NegativeOrZero`` constraint was introduced in Symfony 4.3.
+
+Validates that a value is a negative number or equal to zero. To force that a value
+is only a negative number, see :doc:`/reference/constraints/Negative`.
+To force a value is positive or equal to zero,
+see :doc:`/reference/constraints/PositiveOrZero`.
+
+==========  ===================================================================
+Applies to  :ref:`property or method <validation-property-target>`
+Options     - `groups`_
+            - `message`_
+            - `payload`_
+Class       :class:`Symfony\\Component\\Validator\\Constraints\\NegativeOrZero`
+Validator   :class:`Symfony\\Component\\Validator\\Constraints\\LesserThanOrEqualValidator`
+==========  ===================================================================
+
+Basic Usage
+-----------
+
+The following constraint ensure that:
+
+* the ``withdraw`` of a  bankaccount ``TransferItem`` is a negative number or equal to zero
```

Comment:
I propose to rename `TransferItem` to `BankAccount`, but to be honest, a withdraw where 0 is allowed sounds strange 🤔 Maybe we could find a better example.